.drone/pipelines/default/steps/test/cf_acceptance_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+# shellcheck disable=SC1091
+source ".drone/pipelines/default/runtime/binaries.sh"
+# shellcheck disable=SC1091
+source ".drone/pipelines/default/runtime/config.sh"
+
+# Trigger cf-acceptance-tests.
+bazel run //testing/acceptance_tests
+
+cf_acceptance_tests_pod_name() {
+  kubectl get pods --namespace "${KUBECF_NAMESPACE}" --output name 2> /dev/null | grep "acceptance-tests"
+}
+
+# Wait for cf-acceptance-tests to start.
+wait_for_cf_acceptance_tests_pod() {
+  local timeout="300"
+  until kubectl get pods --namespace "${KUBECF_NAMESPACE}" --output name 2> /dev/null | grep --quiet "acceptance-tests" || [[ "$timeout" == "0" ]]; do sleep 1; timeout=$((timeout - 1)); done
```

Comment:
If `cf_acceptance_tests_pod_name()` uses `--ignore-not-found` we can just call it and check for empty.

Reply:
I'd prefer not changing the logic of these scripts too much in this PR given that they already work and it's mostly a copy over from the Gitlab one. We can improve this script when we move it to be part of the bazel target itself.